src/install/debian/lsc.postinst

```diff
@@ -4,7 +4,8 @@ lsc_logdir=/var/log/lsc
 lsc_user=lsc
 lsc_group=lsc
 
-id ${lsc_user} &>/dev/null || useradd ${lsc_user}
+getent group ${lsc_group} > /dev/null 2>&1 || groupadd -r ${lsc_group}
+getent passwd ${lsc_user} > /dev/null 2>&1 || useradd -r -g ${lsc_group} -d /etc/lsc ${lsc_user}
```

Comment:
a common usage for command line flags is "use the short form when you type in you terminal, use the long form when you write a script". It allows a better readability. Here I think in particular `--system` is really more clear than `-r`